### DISCUSSION TITLE
Certificates for ML Projects capability - Implementation Plan

### DISCUSSION THREAD

**Discussion by priyanka-TL** [Sunbird-Ed/Community#48] [General Discussion]
With reference to the PRD https://docs.google.com/document/d/19NMF7o2eY1dAE0yMBQUbpwp74Jm7sIyFASI_MHBW3tU/edit?usp=sharing and Jira stories https://project-sunbird.atlassian.net/browse/SB-2823, https://project-sunbird.atlassian.net/browse/SB-30209 we have come up with the implementation plan https://project-sunbird.atlassian.net/l/cp/bhj2hoXy 

Kindly help us with your feedback on this 

Note: For some points eg QR code verification, project certificate schema upload location, and asynchronous certificate generation we still need clarity and we are waiting for the responses of other discussion forum threads for the same. We will update the document once we get those queries answered

cc - @aks30 @vijiurs @kiranharidas187 @varsha-sl @Prateek-slokam @aishwaryashikshalokam @VISHNUDAS-tunerlabs @vishwanath1004 @Shakthieshwari

**Comment by priyanka-TL** [comment]
We have completed the pending design items, if someone can review and please provide feedback

**Reply by aks30** [reply]
@sharathkashyap @rajeevsathish @surendrasinghs @rhwarrier
